lib/tasks/pulsearch.rake

```diff
@@ -48,6 +48,7 @@ namespace :pulsearch do
   private
 
     def url_for_file(file)
-      "https://raw.githubusercontent.com/pulibrary/pul_solr/master/solr_configs/orangelight/#{file}"
+      #  "https://raw.githubusercontent.com/pulibrary/pul_solr/master/solr_configs/orangelight/#{file}"
+      "https://raw.githubusercontent.com/pulibrary/pul_solr/cf686290a6e191987e43f5462873f1556763d438/solr_configs/orangelight/#{file}"
```

Comment:
Why attached to a specific commit?

Reply:
This line will be updated as soon as @tampakis updates the orangelight code to work with the new solr changes. Right now the production orangelight is still using the old configs (from the specific commit).

Reply:
@tpendragon I removed the specific commit to avoid confusion. The test will pass as soon as we update the solr configs. Thanks for your comment!